# HG changeset patch
# User Rik <rik@octave.org>
# Date 1516062581 28800
#      Mon Jan 15 16:29:41 2018 -0800
# Node ID a27dcb26f8724dc043ee7f2259a7e07be824cbea
# Parent  6b3c78f84d3b30b34db3e464bb8c2db0ee5fceb5
# Parent  4a4a8b2a5bf2aeca01207d0e0b09bc27da4787d2
maint: merge stable to default.

diff --git a/libinterp/corefcn/regexp.cc b/libinterp/corefcn/regexp.cc
--- a/libinterp/corefcn/regexp.cc
+++ b/libinterp/corefcn/regexp.cc
@@ -1497,9 +1497,13 @@ function.
 
 # Nasty lookbehind expression
 %!test
 %! warning ("off", "Octave:regexp-lookbehind-limit", "local");
 %! assert (regexprep ('x^(-1)+y(-1)+z(-1)=0', '(?<=[a-z]+)\(\-[1-9]*\)', '_minus1'),'x^(-1)+y_minus1+z_minus1=0');
 
 %!assert (regexprep ("\n", '\n', "X"), "X")
 %!assert (regexprep ("\n", "\n", "X"), "X")
+
+## Empty matches were broken on ARM architecture
+%!test <*52810>
+%! assert (strcmp (regexprep ("\nabc", "^(\t*)(abc)$", "$1$2", "lineanchors"), "\nabc"))
 */
